sdk/devcenter/Azure.Developer.DevCenter/tests/EnvironmentsClientTests.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Developer.DevCenter.Tests
+{
+    [PlaybackOnly("As deploy/delete manipulations with real resources take time.")]
```

Comment:
How much time are we talking about here? Are these calls LRO?

Reply:
Create/delete/start/stop operations on Dev Boxes and Environments are LROs. Running the entire test suite for the SDK might take hours (~3-4 hours in my experience).